data/mods/Magiclysm/worldgen/elf_workshop.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "type": "palette",
+    "id": "elf_palette",
+    "terrain": { "3": "t_rdoor_c", "1": "t_strconc_wall", ".": "t_metal_floor_olight" },
+    "furniture": {
+      "a": "f_anvil",
+      "t": "f_table",
+      "b": "f_tablesaw",
+      "c": "f_chair",
+      "d": "f_drill_press",
+      "j": "f_jointer",
+      "r": "f_router",
+      "w": "f_workbench",
+      "m": "f_workbench",
+      "s": "f_stool",
+      "f": "f_demon_forge"
+    },
+    "items": {
+      "a": [
+        { "item": "tools_anvil", "chance": 75, "repeat": [ 1, 2 ] },
+        { "item": "mithril_ingot", "chance": 10 },
```

Comment:
`"items"` can be used for item groups but you'll need to use `"place_loot"` to spawn individual items.

Reply:
This isn't how place_loot works.  It needs to be in the mapgen object entry and it must use x,y coordinates.  item_groups get identified as `group` and single items as `item`.  For example:
```
"place_loot": [ { "item": "scrap_copper", "x": 14, "y": 12, "chance": 100 }, { "group": "tools_anvil", "x": 5, "y": 4, "chance": 100 } ],
```